### ISSUE TITLE
How do I disable validation for AWSProperty types?

### ISSUE THREAD

**Issue by baharev** [cloudtools/troposphere#1820]
The `LambdaFunctionAssociation` does have an [`IncludeBody`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-cloudfront-distribution-lambdafunctionassociation.html#cfn-cloudfront-distribution-lambdafunctionassociation-includebody) property. Yet, when I try to create one with this property:

```
LambdaFunctionAssociation(
        EventType=event_type,
        LambdaFunctionARN=arn,
        IncludeBody=True
)
```

I get:

> AttributeError: LambdaFunctionAssociation object does not support attribute IncludeBody

**How do I disable validation in such cases?**

As a workaround, I first create the object without it, then I assign 

`lfa.properties['IncludeBody'] = True`

It is an acceptable workaround for the time being, but I failed to find the official solution for disabling validation.

**Comment by baharev** [comment]
It's funny: It is currently the last commit that fixes this issue https://github.com/cloudtools/troposphere/commit/50636ecb58827bdaebbaa2abbef9bc2e68d36a84. 

Related: #1819

**Comment by markpeek** [comment]
Fix released in 2.6.4.
